doc/apiary/v2/fiware-ngsiv2-cookbook.apib

```diff
@@ -124,43 +206,79 @@ This resource allows you to obtain a list of subresources published by the API:
         ]
 
 
-### List entities by type list [GET /v2/entities{?type}]
+### By type list [GET /v2/entities{?type}]
 
 + Parameters
-    + type: Room,Car (optional, string) - Type list
+    + type: ParkingLot, StreetParking (optional, string) - Type list
 
 + Response 200 (application/json)
 
         [
          {
-            "type": "Room",
-            "id": "DC_S1_D41",
-            "temperature": 35.6
-         },
-         {
-            "type": "Room",
-            "id": "Boe_Idearium",
-            "temperature": 22.5
+            "type": "ParkingLot",
+            "id": "P12345-A",
+            "name": {
+               "value": "Marques de Pombal",
+               "type": null,
+               "metadata": { }
+            },
+            "availableSpotNumber": {
+              "value": 111,
+              "type": null,
+              "metadata": { }
+            },
+            "totalSpotNumber": {
+              "value": 250,
+              "type": null,
+              "metadata": { }
+            },
+            "lastEntrance": {
+               "value": "2015-11-26T11:45:12.398Z",
+               "type": "date",
+               "metadata": { }
+            },
+            "location": {
+              "type": "geo:point",
+              "value": "40.6378754,-8.6370621",
+              "metadata": {
+                "crs": {
+                  "type":  "urn:ogc:def:crs",
+                  "value": "EPSG::4326"
+                }
+              } 
+            }
          },
          {
-            "type": "Car",
-            "id": "P-9873-K",
-            "speed": {
-                "value": 100,
-                "type": "number",
-                "accuracy": 2,
-                "timestamp": {
-                    "value": "2015-06-04T07:20:27.378Z",
-                    "type": "date"
-                }
+            "type": "StreetParking",
+            "id": "A45678",
+            "availableSpotNumber": {
+              "value": 11,
+              "type": null,
+              "metadata": { }
+            },
+            "totalSpotNumber": {
+              "value": 20,
+              "type": null,
+              "metadata": { }
+            },
+            "location": {
+              "type": "geo:polygon",
+              "value": [
+                "40.64125502310924,-8.653267621994019",
+                "40.64123059990093,-8.652441501617432",
+                "40.64147483158176,-8.650263547897339",
+                "40.64125502310924,-8.653267621994019"
+              ],
+              "metadata": { } 
             }
          }
         ]
 
-### List all entities which id matches a pattern [GET /v2/entities{?idPattern}]
+### By pattern matching [GET /v2/entities{?idPattern,options}]
 
 + Parameters
     + idPattern: DC_* (optional, string) - Only retrieve entities which id matches this pattern.
```

Comment:
DC_*  => DC_.*  ?
Only retrieve entities whose id match this pattern (which => whose)
Now, is it 'match' or 'matches'?
I am not sure, I would avoid the problem by rephrasing:

Only retrieve entities having a matching id.